src/pages/error-page.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { HStack, VStack } from '@chakra-ui/react';
+import { PageContainer, Logo } from '@src/common/components';
+import { ErrorPage } from '@src/modules/auth';
+
+export default function NotifyForgetPasswordSuccess() {
+  const { t } = useTranslation('auth');
+  const head = { title: t('error-page.title') };
+
+  return (
+    <PageContainer head={head}>
+      <VStack flex={1} padding="1rem">
+        <HStack width="100%">
+          <Logo height="5rem" />
+        </HStack>
+        <ErrorPage />
+      </VStack>
+    </PageContainer>
+  );
+}
```

Comment:
For next.js, we can use their default `_error.tsx` page to override it with our design, so that when there are any errors happening, e.g. the page does not exist, it will point us to the `_error.tsx` page.
See this document: https://nextjs.org/docs/advanced-features/custom-error-page

Reply:
issue solved